### ISSUE TITLE
MGI JAX Mouse Phenotypes

### ISSUE THREAD

**Issue by gkos-bio** [opentargets/roadmap#91]
## Description
* [ ] **WHY - What is the problem this feature would answer?**  
>  *Replace this text with a description here. Think of it as the 1-paragraph blog post you would write once the feature is released on the Open Targets platform*

* [ ] **HOW are we going to solve the problem?**  
> *Describe technical design of the solution.*
> *Paste here google doc link or link to markdown file, if already written somewhere else*

* [ ] **WHAT would this look like?**
> *Screenshots and mock-ups are great*
 

## Progress Tracker
- [ ] Description complete
- [ ] Proposal drafted 

### Get to a prototype at branch.targetvalidation.org:
> Write code + **tests** + docs. Isolate the work in feature branches.  
> Use as many PRs as you need.  
> Write tests in the same or different PRs, as is convenient for you.  
> As each PR is merged, add a comment to this issue referencing the PRs.  

- [ ] Feature tag *Code should have a feature tag to be disabled by default.*
- [ ] Minimal functional testing
- [ ] Minimal docs
- [ ] if on webapp, test a deployment on netlify
       
### Push to staging
- [ ] merged code and pushed to staging.targetvalidation.org
  **PULL request should be reviewed by 1-2 people at least**
- [ ] at least 1 week internal testing?
- [ ] received some feedback/input from users (eg. pharma scientist)

### Released
- [ ] in the release notes?
- [ ] documentation updated?		
- [ ] blog post?
- [ ] users that requested it originally informed?

## Make sure you can measure impact
> As dev team we want to know if a new feature are succesful.
> To do that we need to define metrics to assess the impact of each new feature we introduce to the platform
- [ ] metric identified 
- [ ] identify the one metric that matters (OMTM)

http://www.informatics.jax.org/marker/MGI:96559
http://www.informatics.jax.org/diseasePortal/popup?isPhenotype=true&markerID=MGI:96559&header=immune%20system
